### ISSUE TITLE
Font Support for Microsoft TTF Fonts

### ISSUE THREAD

**Issue by kpsychwave** [scrapinghub/splash#243]
Can you add this to the docker file so Microsoft fonts render okay

http://packages.ubuntu.com/trusty/ttf-mscorefonts-installer

**Comment by kmike** [comment]
A good idea, thanks for the suggestion. I'll try it.

**Comment by Youwotma** [comment]
This appears not fixed for me,with this test page:
```
<span style="font-size: 400px; font-family: Arial, sans-serif;">3</span>
<span style="font-size: 400px; font-family: sans-serif;">3</span>
```
hosted [here](http://dev.scrapinghub.com/~david/fonttest.html) for convenience, I get this results: ![results](http://i.imgur.com/maD5NrQ.png)
(left, is rendered by Firefox, chrome or a pyqt4-webkit test browser, right is rendered by splash)

I'm investigating this.

**Comment by kmike** [comment]
@Youwotma did you use qt4 or qt5 based Splash? I've fixed it in py3 / qt5 branch: ttf-mscorefonts-installer package was removed on cleanup stage, and it seems this also removed fonts themseves. So it should work now with qt5-based splash.

In qt4 branch for me these files are not removed - just tried it by pulling "scrapinghub/splash:master" and running your grep command. But for me in qt4 this doesn't work even though necessary files are there.
